caom2pipe/caom2pipe/astro_composable.py

```diff
@@ -0,0 +1,165 @@
+#  any later version.                   toute version ultérieure.
+#
+#  OpenCADC is distributed in the       OpenCADC est distribué
+#  hope that it will be useful,         dans l’espoir qu’il vous
+#  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+#  without even the implied             GARANTIE : sans même la garantie
+#  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+#  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+#  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+#  General Public License for           Générale Publique GNU Affero
+#  more details.                        pour plus de détails.
+#
+#  You should have received             Vous devriez avoir reçu une
+#  a copy of the GNU Affero             copie de la Licence Générale
+#  General Public License along         Publique GNU Affero avec
+#  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+#  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+#                                       <http://www.gnu.org/licenses/>.
+#
+#  $Revision: 4 $
+#
+# ***********************************************************************
+#
+
+import logging
+
+from astropy.coordinates import EarthLocation
+from astropy.time import Time, TimeDelta
+
+from caom2pipe import manage_composable as mc
+
+
+__all__ = ['convert_time', 'get_datetime', 'build_plane_time', 'get_location',
+           'get_timedelta_in_s']
+
+
+def find_time_bounds(headers):
+    logging.debug('Begin find_time_bounds.')
+    date = headers[0].get('DATE-OBS')
+    exposure = headers[0].get('TEXP')
+    return convert_time(date, exposure)
+
+
+def convert_time(start_time, exposure):
+    logging.debug('Begin convert_time.')
+    if start_time is not None and exposure is not None:
+        logging.debug(
+            'Use date {} and exposure {} to convert time.'.format(start_time,
+                                                                  exposure))
+        if type(start_time) is float:
+            t_start = Time(start_time, format='mjd')
+        else:
+            t_start = Time(start_time)
+        dt = TimeDelta(exposure, format='sec')
+        t_end = t_start + dt
+        t_start.format = 'mjd'
+        t_end.format = 'mjd'
+        mjd_start = t_start.value
+        mjd_end = t_end.value
+        logging.debug('End convert_time mjd start {} mjd end {} .'.format(
+            mjd_start, mjd_end))
+        return mjd_start, mjd_end
+    return None, None
+
+
+def get_datetime(from_value):
+    """
+    Ensure datetime values are in MJD.
+    :param from_value:
+    :return: datetime instance
+    """
+    if from_value is not None:
+        try:
+            result = Time(from_value)
+            return result
+        except ValueError:
+            try:
+                # VLASS has a format astropy fails to understand
+                from datetime import datetime
```

Comment:
The local imports are there to avoid collisions between astropy.time.Time, and caom2.Time,
and astropy.time.datetime and datetime. Should I just name the relevant functions with the appropriate package/module names instead?